packages/applications/subatomic-spring-boot-2x-template.yml

```diff
@@ -53,8 +53,10 @@ objects:
         spec:
           containers:
             - name: ${APP_NAME}
-              image: ' ' 
+              image: ' '
               env:
+                - name: SPRING_CLOUD_CONFIG_URI
+                  value: "http://subatomic-config-server.${DEVOPS_NAMESPACE}.svc.cluster.local"
```

Comment:
Do these things work when there is no config server?

Reply:
yup

Reply:
I mean nope